### PR TITLE
Final publication

### PR DESCRIPTION
**Pull request by HolEv**
This PR adds changes relevant for the final publication of the DeepRVAT paper. 

Major changes 
- Addition of code and experiment set-ups for generating supplementary and extended data figures
- Restructuring/renaming of directories
- Update of READMEs

### CODE REVIEW

File: figure3_phenotype_prediction/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The column names of `PRS.parquet` have to be the PGS ids and the index (named `s
 
 ### DeepRVAT gene impairment scores
 The folder `deeprvat_burdens` must have the `burdens.zarr` with the DeepRVAT gene impairment scores for all samples in `train_test_samples`. It also needs the metadata `sample_ids.zarr` and `genes.npy`, which provide the order of samples and genes in `burdens.zarr`. 
-The gene impairment scores are computed with the pre-trained models from `../association_testing/paper_experiment`, which only used unrelated Caucasian samples from the  UKBB 200k WES cohort during training. 
+The gene impairment scores are computed with the pre-trained models from `../figure2_association_testing_benchmark/deeprvat_main_exp`, which only used unrelated Caucasian samples from the  UKBB 200k WES cohort during training. 
 
 ### link to the reference experiment 
 The lists of genes whose gene impairment scores will  be included in the rare variant phenotype predictors are retrieved from the reference experiment `../association_testing/paper_experiment`.
```
Comment (meyerkm) [inline figure3_phenotype_prediction/README.md]: Update link here!

Comment (meyerkm) [reply figure3_phenotype_prediction/README.md]: The file isn't part of the commits, but `timing/README.md` also needs the `../association_testing/paper_experiment` link updated as well.

Comment (HolEv) [reply figure3_phenotype_prediction/README.md]: Done!

**Review by meyerkm — APPROVED** [review]
LGTM !

**Pull request merged**